.storybook/preview.js

```diff
@@ -81,16 +83,19 @@ const themeDecorator = (story, { globals, parameters }) => {
   switch (currentTheme) {
     case THEMES.NG_LIGHT:
       styleElement.textContent = clrUiStyles;
+      document.body.removeAttribute('cds-text');
       document.body.removeAttribute(cdsThemeAttribute);
       document.body.style.backgroundColor = getClrUiAppBackgroundColor(currentTheme);
       break;
     case THEMES.NG_DARK:
       styleElement.textContent = clrUiDarkStyles;
+      document.body.removeAttribute('cds-text');
       document.body.removeAttribute(cdsThemeAttribute);
       document.body.style.backgroundColor = getClrUiAppBackgroundColor(currentTheme);
       break;
     default:
       styleElement.textContent = `${clrUiStyles}${cdsCoreAndShimStyles.join('')}`;
+      document.body.setAttribute('cds-text', 'body');
```

Comment:
Should we update the "@cds/core shim" docs page to include this? Currently, we say the shim only affects color, but this makes our storybook shim usage affect typography as well.

Reply:
And our stories currently don't use `cds-text` typography where appropriate, so I am afraid that this may result in some odd typography that we don't actually want.

Reply:
I'll update the shim docs to add cds-text to the body. 

Until we get completely migrated in v17, our components won't use the cds-text everywhere they should be using them but this is a decent enough workaround since the core components inherit a lot of styling from the page.

Reply:
Okay, that sounds fine to me!